### PR TITLE
Remake with Flexbox and Grid

### PR DESCRIPTION
**Pull request by Lorenzo-Mariano**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-let part1 = document.querySelector('.part1')
+let userInput = document.querySelector('.user-input')
 let part2 = document.querySelector('.part2')
```
Comment (mikolertesx) [inline app.js:2]: You should probably change this name.
part2 is not very intuitive.
You can do an f2, and change the name to whatever else.

File: style.css [hunk 2]
```diff
@@ -18,48 +24,86 @@ body {
...
+.input-space {
+  display: flex;
+  flex-direction: column;
+
+  height: 100px;
+  border-radius: inherit;
+}
+
+.user-input {
+  /* grid-column: span 4; */
```
Comment (mikolertesx) [inline style.css:49]: Probably remove the comment.

File: style.css [hunk 3]
```diff
@@ -18,48 +24,86 @@ body {
...
   position: absolute;
   width: 300px;
   margin-left: 18px;
-}
-
-.make-block {
-  display: inline-block;
-  width: auto;
-}
+} */
```
Comment (mikolertesx) [inline style.css:106]: Same thing.

File: style.css [hunk 4]
```diff
@@ -116,8 +160,8 @@ button:active {
 .equal {
   border-color: #75b1ff;
   height: 156px;
-  position: absolute;
-  right: 44px;
+  /* position: absolute;
+  right: 44px; */
```
Comment (mikolertesx) [inline style.css:164]: Same thing.

**Review by mikolertesx — CHANGES_REQUESTED** [review]

File: style.css (hunk 4 above)
Comment (Lorenzo-Mariano) [reply style.css]: aight got you

**Pull request merged**